content/events/hacktoberfest/faq.adoc

```diff
@@ -123,9 +123,10 @@ You need to be a member of a respective GitHub organization to send such request
 
 Unfortunately there is no way to get organization-wide metrics for Hacktoberfest,
 because the event organizers do not share info about registered users.
-This is why we recommend to mark pull request with `hacktoberfest` label or "Hacktoberfest" in the title.
+This is why we recommend to mark pull requests with `hacktoberfest`, `hacktoberfest-accepted` label or "Hacktoberfest" in the title.
```

Comment:
cc @jmMeessen @MarkEWaite

Reply:
Sorry @NotMyFault to have a bit derailed your PR, the explanation about the later addition of the label should be mentioned somewhere indeed.

Reply:
> I think the recommendation to add these labels on pull requests should be done only for those in repositories not tagged, it's noisy and useless otherwise.

The label(s) come in handy to distinguish between regular PRs, such as dependabot/renovate and miscellaneous chores and actual hacktoberfest items, like feature additions, bug fixes, etc.
Especially in core repositories, I found this handy, considering not every PR targeting a repository with the `hacktoberfest` topic necessarily is a countable hacktoberfest contribution.

> Sorry @NotMyFault to have a bit derailed your PR

I think, I'll survive that :P

Reply:
> Exclude PRs with label "dependencies", this condition should catch all Dependabot/Renovabot/Updatecli PRs

Note that some dependency updates are done manually, e.g. when it is not just bumping the version. I would rather recommend to avoid it and to filter out dependabot if needed